src/app/containers/App/App.jsx

```diff
@@ -15,14 +15,19 @@ export const App = ({ routes, location, initialData, bbcOrigin, history }) => {
     errorCode,
     route: { pageType },
   } = getRouteProps(routes, location.pathname);
-
   const { pageData, status, error, timeOnServer } = initialData;
 
+  const pageLang = pageData.metadata.language;
```

Comment:
As another alternative to doing this in the `App.jsx`, we could check the `pageType` and `service` in [this file](https://github.com/bbc/simorgh/blob/0ba61e622c138a99076c7ae8ae3ea4d95e65cc24/src/app/containers/PageHandlers/withContexts.jsx#L1) to apply the right variant.